src/systemtest/java/com/ericsson/ei/systemtest/testexecutionflow/TestExecutionFlowSteps.java

```diff
@@ -0,0 +1,105 @@
+package com.ericsson.ei.systemtest.testexecutionflow;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.support.AbstractTestExecutionListener;
+
+import com.ericsson.ei.systemtest.utils.Config;
+import com.ericsson.ei.systemtest.utils.StepsUtils;
+import com.ericsson.eiffelcommons.utils.ResponseEntity;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+@Ignore
+public class TestExecutionFlowSteps extends AbstractTestExecutionListener{
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestExecutionFlowSteps.class);
+    private static final String JENKINS_TOKEN = "123";
+    private static final String JENKINS_JOB_XML = "jenkinsJobTemplate.xml";
+
+    private ArrayList<String> jenkinsJobNames = new ArrayList<String>();
+    private Config config = new Config();
+
+    @Given("^configurations are provided$")
+    public void configurations_are_provided() {
+        config.initEIFrontend();
+        config.initEIBackend();
+        config.initJenkinsConfig();
+        config.initRemRemConfig();
+    }
+
+    @Given("^a jenkins job '\\\"([^\\\"]*)\\\"' from '\"([^\"]*)\"' is created$")
+    public void a_jenkins_job_from_is_created(String jenkinsJobName, String scriptFileName) throws Throwable {
+        boolean success = StepsUtils.createJenkinsJob(
+                jenkinsJobName,
+                scriptFileName,
+                config.getJenkinsBaseUrl(),
+                config.getJenkinsUsername(),
+                config.getJenkinsPassword(),
+                JENKINS_TOKEN,
+                JENKINS_JOB_XML,
+                config.getRemremBaseUrl()
+         );
+
+        if (success) {
+            jenkinsJobNames.add(jenkinsJobName);
+        }
+
+        assertTrue("Failed to create jenkins job.", success);
+    }
+
+    @Then("^we continue with the next step$")
+    public void we_continue_with_the_next_step() {
+        //Just for cucumber to make sense
+    }
+
+    @Then("^subscriptions and jenkins jobs should be removed$")
+    public void subscriptions_and_jenkins_jobs_should_be_removed() throws Throwable {
+        StepsUtils.deleteJenkinsJobs(jenkinsJobNames);
+        StepsUtils.deleteSubscriptions(config.getEiFrontendBaseUrl(), config.getEiBackendBaseUrl());
+    }
+
+    @Given("^subscription object \"([^\"]*)\" is created which will trigger \"([^\"]*)\"$")
+    public void subscription_is_created(String subscriptionName, String nameOfTriggeredJob) throws Throwable {
```

Comment:
"nameOfTriggeredJob" ... this sounds like the job has been triggered already. Why not use name such as, e.g: "jenkinsJobName"

Reply:
Everything in this file you just approved by you @tobiasake in the artifact flow but it is a fair point. How should we handle this now that the other one is merged. Should you @Christoffer-Cortes update it in here and then i will make the same update in my PR on the artifact flow?